x-pack/legacy/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -6,21 +6,46 @@
 import fs from 'fs';
 import os from 'os';
 import path from 'path';
+// @ts-ignore
 import puppeteer from 'puppeteer-core';
 import rimraf from 'rimraf';
 import * as Rx from 'rxjs';
 import { map, share, mergeMap, filter, partition } from 'rxjs/operators';
+import { InnerSubscriber } from 'rxjs/internal/InnerSubscriber';
+
 import { HeadlessChromiumDriver } from '../driver';
-import { args } from './args';
+import { args, IArgOptions } from './args';
 import { safeChildProcess } from '../../safe_child_process';
 import { getChromeLogLocation } from '../paths';
 
-const compactWhitespace = str => {
+type binaryPath = string;
+interface ILogger {
+  isVerbose: boolean;
+  warning: (msg: string) => any;
+  debug: (msg: string) => any;
+  error: (msg: string) => any;
+}
```

Comment:
I think we can import the definition from `x-pack/legacy/plugins/reporting/types.d.ts`